test/queue.js

```diff
@@ -0,0 +1,86 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Queue from '../src/Queue'
+
+chai.use(chaiChange)
+  let queue
+
+describe('Queue', function () {
+  beforeEach(function(){
+    queue = new Queue()
+  }) 
+
+  it('exists', function () {
+    expect(Queue).to.be.a('function')
+  })
+
+  context('enqueue()', function () {
+
+    it('adds an element to the back of the queue', function () {
+      queue.enqueue('dude')
+      queue.enqueue('bro')
+      queue.enqueue('foo')
+
+      expect(queue.back()).to.deep.equal('foo')
+    })
+
+    it('increases the queue size', function () {
+      queue.enqueue(10)
+      queue.enqueue(11)
+      queue.enqueue(12)
+
+      expect(queue.length()).to.equal(3)
+    })
+  })
+
+  context('dequeue()', function () {
+    it('returns and removes the front element in the queue', function() {
+      queue.enqueue(12)
```

Comment:
I tested this with `
 queue.enqueue(10)
 queue.enqueue(11)
 queue.enqueue(12) ` and it worked, of course. I think the test could benefit by enqueuing multiple elements so that we know `dequeue()` always returns the front element, and not just the only element.